test/get-core/edges.spec.js

```diff
@@ -19,7 +19,9 @@ describe('Edges', function() {
                     videos: {}
                 }
             },
-            cache: cacheGenerator(0, 1)
+            cache: cacheGenerator(0, 1),
+            requestedMissingPaths: [],
+            optimizedMissingPaths: []
```

Comment:
This tests for those spurious `"undefined"` missing paths. Prior to this bugfix, these arrays would contain `[["videos", "undefined", "title"]]`.